agreementbot/secure_api.go

```diff
@@ -1,14 +1,30 @@
-// @APIVersion 1.0.0
-// @APITitle Agreement Bot Secure API
-// @APIDescription This is the secure API for the agreement bot.
-// @BasePath https://host:port/
-// @SubApi Deployment Check API [/deploycheck]
-
+// Package agreementbot Agreement Bot Secure API
```

Comment:
Please update the README with the command to generate the swagger, and include the swagger json in the PR.